demos/projects/ESPRESSIF/az-ca-recovery/components/sample-azure-iot-ca-recovery/CMakeLists.txt

```diff
@@ -41,4 +44,4 @@ set(COMPONENT_INCLUDE_DIRS
 idf_component_register(
     SRCS ${COMPONENT_SOURCES}
     INCLUDE_DIRS ${COMPONENT_INCLUDE_DIRS}
-    REQUIRES mbedtls tcp_transport coreMQTT azure-sdk-for-c azure-iot-middleware-freertos)
+    REQUIRES nvs_flash mbedtls tcp_transport coreMQTT azure-sdk-for-c azure-iot-middleware-freertos)
```

Comment:
Documentation: 
1. Could we add some documentation on the SDK-reserved NVS key names and what they will contain on the device?
2. We should strongly advise NVS cryptography for the trust bundle. Anyone that can write to NVS the trust bundle (or the firmware trust bundle in our original samples) can gain full control to the TLS channel. (NVS or firmware encryption is out of scope for these samples.)